src/marlin_stubs/pause/pause.cpp

```diff
@@ -651,18 +742,26 @@ void Pause::park_nozzle_and_notify() {
     //home the X or Y axis if it is not homed and we want to move it
     //homing is after Z move to be clear of all obstacles
     //Should not affect other operations than Load/Unload/Change filament run from home screen without homing. We are homed during print
-    if ((axis_homed & _BV(X_AXIS)) == 0 && current_position.x - park_pos.x != 0) {
+    if (!isnan(park_pos.x) && (axis_homed & axis_known_position & _BV(X_AXIS)) == 0) {
```

Comment:
If we change this it will break functionality of M1400 beacuse it sets park_pos to current pos if it does not want to move.